index.html

```diff
@@ -1264,11 +1264,42 @@
       </section>
 
       <section>
-        <h3>Storage Location of Claims</h3>
+        <h3>Storage Providers and Data Mining</h3>
 
-        <p class="issue">
-Storage location of claims may mine data or forward data.
+        <p>
+When a holder receives a claim from an issuer, it is highly likely that the
+claim will need to be stored somewhere (e.g. in a credential repository).
+Holders are warned that the information in a verifiable claim is often
+sensitive in nature and highly individualized, making it a high value
+target for data mining. It is expected that there will be many services
+that store verifiable claims for free and mine personal data and sell it
+to organizations that desire individualized profiles on people and
+organizations (i.e. if the service is free, you are the product).
```

Comment:
Let's not say "it is expected" but we can say "Therefore, there may be services..."